src/Dapr.Actors/ActorId.cs

```diff
@@ -24,7 +24,11 @@ public class ActorId
         /// <param name="id">Value for actor id.</param>
         public ActorId(string id)
         {
-            this.stringId = id ?? throw new ArgumentNullException(nameof(id));
+            if (string.IsNullOrEmpty(id))
```

Comment:
Thanks for fixing this.  This can be made stronger with IsNullOrWhiteSpace().

Reply:
Yes but I did not know if white space ids where supported or not. If not, I can make the change if you want.

Reply:
They are not supported, please make the suggested change.

Reply:
ok

Reply:
done

Reply:
Much appreciated.